### PR TITLE
basic frame work updated

### PR DESCRIPTION
**Pull request by alwaysjaden**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -27,15 +43,39 @@ <h5>Mike Mikerson</h5>
...
+                  <input type="submit" class="button secondary" value="Search">
+                </div>
+              </div>
+              
+              </div>
+              
+                
+              
             </div>
-          </div>
```
Comment (cnix273) [inline index.html:35]: Hey Jaden! Can you check to see if this div closure removal was necessary or not?

**Review by cnix273 — CHANGES_REQUESTED** [review]
1 change in line 35 needs to be reviewed

File: index.html (hunk 1 above)
Comment (alwaysjaden) [reply index.html:35]: YES IT WAS

**Review by cnix273 — APPROVED** [review]
Sounds goos! Approving the updated files

**Comment by cnix273** [comment]
Merging request to update html and css

**Pull request merged**